src/lib/bootloader/sections.rb

```diff
@@ -36,7 +36,7 @@ def default=(value)
       log.info "set new default to '#{value.inspect}'"
 
       # empty value mean no default specified
-      raise "Unknown value #{value.inspect}" if !all.include?(value) && !value.empty?
+      raise "Unknown value #{value.inspect}" if !all.empty? && !all.include?(value) && !value.empty?
```

Comment:
well, I do not like using unless if it is more then one condition as it is harder to see what it actually is as using unless is same as `if !(all.empty? || all.include?(value) || value.empty?)` which at least for me need some brain processing.